### PR TITLE
feat(variable): add description field

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

Peer programming @fabienfleureau 
https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-725

Screenshot 2024-07-04 at 15 58 01

Screenshot 2024-07-04 at 15 58 12

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] my-env - feat(variable): add description field - 2024-07-04T13:58:59Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/c3f125a2-1f09-47d0-82bc-e28f7f760a98/services/general)

 Another comment will be posted when deployments are finished

File: libs/domains/variables/feature/src/lib/variable-list/variable-list.tsx [hunk 1]
```diff
@@ -199,39 +199,49 @@ export function VariableList({
...
-                )}
-                <Tooltip align="start" content={variable.key || ''}>
-                  <span className="truncate text-sm font-medium">{variable.key}</span>
-                </Tooltip>
-                {variable.owned_by === ExternalServiceEnum.DOPPLER && (
-                  <Tooltip content="Sync with Doppler">
-                    <span className="ml-2">
-                      <Icon name={IconEnum.DOPPLER} width="11px" height="11px" />
+              <div className="flex items-center gap-2">
+                <div className="truncate">
```
Comment (RemiBonnet) [inline libs/domains/variables/feature/src/lib/variable-list/variable-list.tsx:206]: I added missing truncate for naming

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**